bench/index.html

```diff
@@ -25,6 +30,13 @@
 </head>
 
 <body class="prose">
+    <div id="benchmarks">
+        <a href='/bench/buffer' class='button'>Buffer</a>
+        <a href='/bench/fps' class='button'>FPS</a>
+        <a href='/bench/query-box' class='button'>Query Box</a>
+        <a href='/bench/query-point' class='button'>Query Point</a>
+        <a href='/bench/geojson-rendering' class='button'>GeoJSON Rendering</a>
+    </div>
```

Comment:
I'm :-1: on a manually-maintained list of benchmarks. We should either generate this automatically or omit it altogether.